source/expectations/data_values/ut_cursor_details.tpb

```diff
@@ -146,13 +146,25 @@ create or replace type body ut_cursor_details as
 
   member function get_missing_join_by_columns( a_expected_columns ut_varchar2_list ) return ut_varchar2_list is
     l_result ut_varchar2_list;
+    l_prefix            varchar2(125);
   begin
+      if self.is_anydata = 1 then
+        l_prefix := get_root;
```

Comment:
I would rather call it `l_root` everywhere.
It took me a while to understand what is the meaning of `l_prefix`.